shell.nix

```diff
@@ -22,4 +22,8 @@ in
       pkgs.elmPackages.elm-format
 
     ];
+
+    shellHook = ''
+      ${pkgs.just}/bin/just install-deps
+    '';
```

Comment:
The main change ☝️

The multiple-single-quote syntax ( `''''`)is a multiline string. You can write in here like a regular shell script.